### PR TITLE
HC Dashboard - Prenatal

### PR DESCRIPTION
**Pull request by anvmn**
#966
#971

### CODE REVIEW

**Comment by anvmn** [comment]
Selection_029

**Comment by anvmn** [comment]
Selection_031

**Comment by anvmn** [comment]
Selection_032

**Comment by anvmn** [comment]
Selection_036

**Comment by anvmn** [comment]
Selection_046

File: client/src/elm/Pages/Dashboard/View.elm [hunk 1]
```diff
@@ -1027,36 +1031,144 @@ viewPrenatalPage language currentDate isChw assembled db model =
...
+viewPrenatalDiagnosesSection : Language -> NominalDate -> Int -> List PrenatalDataItem -> List (Html Msg)
+viewPrenatalDiagnosesSection language currentDate currentlyPregnantWithDangerSigns data =
+    let
+        totalHighRiskPregnancies =
+            newlyDiagnosesSyphilisPregnancies
+                ++ newlyDiagnosesPreeclampsiaPregnancies
+                ++ newlyDiagnosesEclampsiaPregnancies
+                ++ newlyDiagnosesSevereAnemiaPregnancies
+                ++ newlyDiagnosesAcuteMalnutritionPregnancies
+                ++ newlyDiagnosesAcuteMalnutritionPregnancies
```
Comment (mariano-dagostino) [inline client/src/elm/Pages/Dashboard/View.elm]: Is this repeated? @anvmn

**Review by mariano-dagostino — CHANGES_REQUESTED** [review]
@anvmn Looks good, but there is one thing to check.

File: client/src/elm/Pages/Dashboard/View.elm (hunk 1 above)
Comment (anvmn) [reply client/src/elm/Pages/Dashboard/View.elm]: Duplicate indeed. Nice catch @mariano-dagostino

**Pull request merged**